client/src/app/schema/profile.ts

```diff
@@ -27,4 +27,6 @@ export interface Profile {
 
   avatarType: string;
   avatarUrl: string;
+
+  hasPayments: boolean | null;
```

Comment:
why is this nullable?

Reply:
Because not every endpoint that returns Profile data sets this field. (It requires additional DB query so I did not want to add it everywhere.)